### DISCUSSION TITLE
Uninstall docker-slim

### DISCUSSION THREAD

**Discussion by nopria** [slimtoolkit/slim#449] [Q&A]
Since [I can't update docker-slim](https://github.com/slimtoolkit/slim/discussions/421#discussion-4509399) from version 1.38.0, I would like to uninstall in order to make a brand-new install, but I couldn't find any info about that, maybe I can just delete the containing directory?

Thanks.

**Comment by kcq** [comment]
Deleting the slim binaries should be enough (`docker-slim` and `docker-slim-sensor` or `slim` and `slim-sensor` if you have the newer version installed).

There's a data directory you can also remove (`/tmp/docker-slim-state` or `/tmp/slim-state`). And there's also the sensor and state volumes that you might also delete if you want to be thorough (do `docker volume ls` and there you'll see `docker-slim-sensor.X.Y.Z` and `docker-slim-state` or `slim-sensor.X.Y.Z` and `slim-state`).
